portal-api/src/v1/controllers/utilisation-report-service/previous-reports.controller.js

```diff
@@ -39,15 +39,21 @@ const getPreviousReportsByBankId = async (req, res) => {
   try {
     const { bankId } = req.params;
 
-    const { data } = await api.getUtilisationReports(bankId);
+    const { status, data } = await api.getUtilisationReports(bankId);
+
+    if (status !== 200) {
+      console.error('Unable to get previous reports');
+      return res.status(500).send(data);
+    }
+
     const years = getYears(data);
     const groupedReports = getReportsGroupedByYear(years, data);
     const reportsGroupedByYear = populateOmittedYears(groupedReports, years);
 
-    res.status(200).send(reportsGroupedByYear.sort((a, b) => b.year - a.year));
+    return res.status(200).send(reportsGroupedByYear.sort((a, b) => b.year - a.year));
   } catch (error) {
     console.error('Unable to get previous reports %s', error);
```

Comment:
I think we could remove `%s` to potentially get more information about the error here (the entire object instead of just the `error.toString()`. We could change this to `%O` to get the full object, but I would just remove the placeholder completely, i.e.
```
console.error('Unable to get previous reports:', error);
```
which will do pretty much the same thing.